recipes/slncky/meta.yaml

```diff
@@ -0,0 +1,34 @@
+{% set version = "1.0.3" %}
+
+package:
+    name: slncky
+    version: {{ version }}
+
+source:
+  fn: slncky-{{ version }}.tar.gz # [unix]
+  url: https://github.com/slncky/slncky/archive/{{ version }}.tar.gz # [unix]
+  md5: b8a1b22acd355851d18a40fb06e7f510 # [unix]
+
+build:
+    number: 0
+
+requirements:
+    build:
+        - python >=2.7,<3
```

Comment:
please skip the build in the build section for py3k and remove the pinning here.

Reply:
ok.